 hugegraph Pulling 
 hugegraph Error failed to resolve reference "docker.io/hugegraph/hugegraph:1.4.2": docker.io/hugegraph/hugegraph:1.4.2: not found
Error response from daemon: failed to resolve reference "docker.io/hugegraph/hugegraph:1.4.2": docker.io/hugegraph/hugegraph:1.4.2: not found
